src/nvim/api/vim.c

```diff
@@ -289,7 +289,7 @@ ArrayOf(String) nvim_list_runtime_paths(void)
 ///
 /// @param dir The new working directory
 /// @param[out] err Details of an error that may have occurred
-void nvim_change_directory(String dir, Error *err)
+void nvim_set_current_dir(String dir, Error *err)
```

Comment:
Can be fixed in a later PR, but API users would expect to also have `nvim_get_current_dir`, by symmetry or something.